### DISCUSSION TITLE
联机功能回归！！！

### DISCUSSION THREAD

**Discussion by laoliuMC** [Meloong-Git/PCL#5340] [提问]
快点回归以前的联机功能！！！

**Comment by Ad-closeNN** [comment]
![image](https://github.com/user-attachments/assets/8976d197-d001-4fce-a65a-7b58063b3698)
https://github.com/Hex-Dragon/PCL2/discussions/1930

**Comment by Silverteal** [comment]
要回归联机功能，您可以向我支付50元，我会在收到款项后一周内购买疯狂星期四，感谢您的支持。

**Comment by DonDroble** [comment]
不看这个我都忘了之前有过联机功能hhh
